src/components/views/messages/MessageActionBar.tsx

```diff
@@ -325,6 +325,19 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
                     />);
                 }
             }
+            // Show thread icon even for deleted messages, but only within main timeline
+            if (this.context.timelineRenderingType === TimelineRenderingType.Room &&
```

Comment:
```suggestion
            if (this.context.timelineRenderingType !== TimelineRenderingType.Thread &&
```

I believe the above would probably be more desirable. Imagine having a file, and a user creates a thread by replying to that event. It will be shown in the `FilePanel` and the `RoomView`.

The former should have a `TimelineRenderingType` of `File` and the latter of `Room`. But both should have the action to jump to the thread

Reply:
@gsouquet maybe then we should add proper rendering context to the different usecases instead, so that Room means actually main timeline, and not File or Notification?

Reply:
Yeah sounds good 👍